site/docs/.vitepress/theme/style/_fonts.scss

```diff
@@ -1,74 +1,18 @@
-/* red-hat-display-700 - latin-ext_latin */
 @font-face {
-  font-display: swap;
-  font-family: "Red Hat Display";
-  font-style: normal;
-  font-weight: 700;
-  src: url("/fonts/red-hat-display-v14-latin-ext_latin-700.woff2")
-    format("woff2");
-}
-/* noto-sans-display-700 - cyrillic-ext_cyrillic */
-@font-face {
-  font-display: swap;
```

Comment:
So we now use block not swap? Can you explain?

Reply:
No, we’re still using swap. This `font-display` thing is in another line now. See the diff below on the right side.

Reply:
I found it